dom.bs

```diff
@@ -7604,17 +7603,12 @@ are:
  <li><p>Let <var>registry</var> be <a>this</a>'s <a>node document</a>'s
  <a for=Document>custom element registry</a>.
 
- <li>
-  <p>If <var>init</var>["{{ShadowRootInit/customElementRegistry}}"] is non-null:
+ <li><p>Set <var>registry</var> to <var>init</var>["{{ShadowRootInit/customElementRegistry}}"].
```

Comment:
Actually this doesn't work. It needs to be conditional on customElementRegistry existing.